### ISSUE TITLE
Unknown client name

### ISSUE THREAD

**Issue by NolanBecker** [justinemter/pseudo-channel#50]
I'm just confused on connecting it to the client. I have a raspberry pi setup with RasPlex and when I try to run PseudoChannel.py it tells me "Unknown client name: RasPlex". The System Name of my rasplex is "RasPlex".
Can you tell me how to set up the client so that PseudoChannel can find it?

Thanks,
Nolan

**Comment by irodimus** [comment]
Try running PseudoChannel.py -c and see what clients show up. If your client doesn't show there, that could cause your issue.

**Comment by justinemter** [comment]
If that doesn't work, you may not have authenticated your RasPlex with your Plex Server... make sure you can use your RasPlex device as-is to play your Plex content outside of PseudoChannel. That'll ensure that it is recognized.

**Comment by NolanBecker** [comment]
I've authenticated it and connected to my Plex Server and can watch videos without any problem, but when I run PseudoChannel.py -c nothing shows up. I can try reinstalling RasPlex completely, maybe that will help.

**Comment by justinemter** [comment]
hmm, it might be the `plex_token.py` file... have you moved the `plex_token.py` from the project directory up one level so it sits just outside the `./pseudo-channel/` dir? Or if you try and run `python PseudoChannel.py -u` it should throw an error that it cannot find the plex_token.py file and/or it cannot connect to your Plex lib.

**Comment by justinemter** [comment]
@NolanBecker @irodimus  I've been tweaking the xml today and committing my changes to the master repo... I'd hate for anyone to pull the latest commit to find that their xml has been overwritten. Just wanted to give you a heads up!

**Comment by NolanBecker** [comment]
Thanks for the heads up!
My plex_token.py file is in the right place and seems to be working correctly. It updates the database and is able to connect to my plex server and find shows without any problem so I don't think that's the issue. I'm re-writing RasPlex to my raspberry pi sd card right now, so I will see if that fixes it.

**Comment by justinemter** [comment]
Ok great. I remember re-naming my RasPlex a while ago and there are actually 2 places where you name it in settings/preferences.. 1 of the places affects the actual name as seen by Plex, and the other - I don't really know. I think I used this thread to fix the issue for me: https://github.com/RasPlex/RasPlex/issues/174. Scroll to the bottom to find the solution I used... this helped me change the default name from "RasPlex" to "RasPlex2". This may or may not help.
